basics/a68.go

```diff
@@ -0,0 +1,18 @@
+//maps: Deleting values from maps
+
+package main
+
+import "fmt"
+
+func main() {
+	ages := make(map[string]int)
+	ages["alice"] = 31
+	ages["charlie"] = 34
+	fmt.Println("map ages:", ages)
+	delete(ages, "slice")
+	fmt.Println("after deleting slice", ages)
+	// check what happens when we try to delete a key which doesn't exit
+	// so no error is thrown out when trying to delete a key which doesn't exist
```

Comment:
Can you check if a key exists and print if it exists or not?

Reply:
Updated the code